### DISCUSSION TITLE
New IKEA device for PM2.5 Temp, Humitity and VOC, Matter ready. VINDSTYRKA

### DISCUSSION THREAD

**Discussion by trlafleur** [arendst/Tasmota#17957] [General]
VINDSTYRKA

https://9to5mac.com/2023/02/14/ikea-smart-indoor-air-quality-monitor-matter-homekit/

Measures:
    Particle matter (PM2.5)
    Humidity
    Temperature
    Total volatile organic compound levels (TVOC)

IKEA has shared with us that the DIRIGERA smart hub doesn’t have Matter support yet (after its [launch last fall](https://9to5mac.com/2022/10/24/ikea-dirigera-smart-home-hub-matter-november-launch/)), but it is “ready” to enable it with a future update. As noted below, the new air quality monitor does not have HomeKit or Matter capability on its own but can gain it when paired with the DIRIGERA but only after IKEA pushes the Matter update to the hub. We’ve reached out to IKEA asking about a Matter launch date.

**Comment by barbudor** [comment]
Paired how?
Is it zigbee?

I have a vindriktning in which I put an esp8266 + cs811 + hdc1080 and running Tasmota. Does all the same for a fraction of the price

**Reply by host-felice** [reply]
well, speaking of cost, you don't really need vindriktning nor a VINDSTYRKA :-p just add a pm 2.5 sensor and you are done :-D The only reason to buy them is the case (vindriktning) and the lcd (VINDSTYRKA), but people should consider that adding an esp inside that case will alter the temperature and the humidity readings

Yes, VINDSTYRKA is zigbee, but it sends temp without decimals,but you can hook up an esp and tasmota is ready to use it and it will send decimals: https://github.com/trlafleur/Tasmota-VINDSTYRKA?utm_source=pocket_reader

Again for the temperature issue: maybe it is better to hook the esp outside in a small case?

**Comment by Sysselix** [comment]
> Again for the temperature issue: maybe it is better to hook the esp outside in a small case?

Or you could put the temperature-sensor outside the housing: e.g. mounting a DHT22
I just put an SCD41 (Co² + Temp + Hum + DEW) in the housing. Works fine, too :-)

And if you have offsets: just set them with `TempOffset`

**Reply by sfromis** [reply]
Using `TempOffset` is no "solution" to misplacing a sensor near a "radiator" producing heat. The difference can neither be expected to be stable nor the same across a temperature range.

Normally it would be preferable to have the heat source at the top and the sensor at the bottom (if not too close), and ensuring airflow. Using deep sleep can also be relevant, if you do not need frequent values.
